composer.json

```diff
@@ -1,24 +1,19 @@
 {
-  "name": "guiwoda/route-binder",
-  "description": "Laravel route binding, done right.",
-  "autoload": {
-    "psr-4": {
-      "GuiWoda\\RouteBinder\\": "src/"
-    }
-  },
-  "autoload-dev": {
-    "psr-4": {
-      "Tests\\": "tests/"
-    }
-  },
-  "license": "MIT",
-  "require": {
-    "illuminate/support": "~4.2",
-    "illuminate/config": "~4.2",
-    "illuminate/routing": "~4.2"
-  },
-  "require-dev": {
-    "phpunit/phpunit": "~4.4",
-    "mockery/mockery": "0.9.*"
-  }
+    "name": "guiwoda/route-binder",
+    "description": "Laravel route binding, done right.",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": {
+            "GuiWoda\\RouteBinder\\": "src/"
+        }
+    },
+    "require": {
```

Comment:
Why is this change pushed? I can't see why you'd remove the required `config` and `routing` packages